src/analytics/selectors.ts

```diff
@@ -51,6 +54,20 @@ export const getCurrentUserTraits = createSelector(
       }
     }
 
+    const { superchargingTokenConfig, hasMaxBalance } = balanceInfoForSupercharge
```

Comment:
With the new new changes you can get a  `superchargeUsdBalance` from the `superchargeInfoSelector`,

Reply:
oh yeah my branch was behind your one. think it's up to date now!